.gitignore

```diff
@@ -1,11 +1,12 @@
-/.phpunit.result.cache
-build/
-logs/
-reports/
-dist/
-download/
-vendor/
-/*.phpunit.xml
-/phpunit.xml
-/.phpcs-cache
-/phpstan.neon
+/.phpunit.result.cache
+.idea/

```

Comment:
Please drop that commit. I'd rather not open the door to adding rules for every editor out there. Instead, developers should globally ignore their editor directory if any once in their life and be done with it.

Reply:
Thanks, @greg0ire I'll do that.

Unrelated to this, is there a way to run the integration tests locally? The test I've added does return the expected `varchar` on a database I have with this version of the driver, so I'm not sure what's going wrong here.

Reply:
There is no easy, documented way to run all tests for all databases if that's what you want, but you can still look at `.travis.yaml`, pick and job and runs what it runs.

Reply:
Thanks @greg0ire - I think I've fixed the issue (misunderstood what `fetchOne` returns).

Maybe in a subsequent PR we could add a docker-compose configuration to make it easier to run integration tests locally?

Reply:
> Maybe in a subsequent PR we could add a docker-compose configuration to make it easier to run integration tests locally?

IIRC, that kind of proposal is rejected on grounds that it it does not also run in the CI, it's going to break without us noticing. But I think some documentation on this, maybe in CONTRIBUTING.md would be welcome